src/vm/i386/jithelp.asm

```diff
@@ -1464,4 +1464,35 @@ JIT_EndCatch PROC stdcall public
 
 JIT_EndCatch ENDP
 
+; The following helper will consequently access ("probe") a word on each page of the stack
```

Comment:
nits about your comments:
1. remove the word "consequently"; it is not necessary (I think you meant "subsequently" instead).
2. change "[esp]" to "esp" (or surround it with single-quotes). As is, it looks like a dereference of esp.

Reply:
Thanks for the additional feedback! This has been addressed as well.